src/components/FallbackAlert.messages.jsx

```diff
@@ -0,0 +1,22 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  'ecommerce.order.history.fallback.alert.title': {
+    id: 'ecommerce.order.history.fallback.alert.title',
+    defaultMessage: 'Something went wrong.',
+    description: 'Title for fallback error message',
+  },
+  'ecommerce.order.history.fallback.alert.body': {
+    id: 'ecommerce.order.history.fallback.alert.body',
+    defaultMessage:
+      'Refresh this page and try again. If this problem persists, {supportLink}.',
+    description: 'Body for fallback error message',
```

Comment:
change it to
description: 'Description for fallback error message with the support link.'